tests/servicebus_functions/get_servicebus_triggered/function.json

```diff
@@ -0,0 +1,23 @@
+{
+  "scriptFile": "__init__.py",
+  "disabled": false,
+  "bindings": [
+    {
+      "type": "httpTrigger",
```

Comment:
serviceBusTrigger?

Reply:
ServiceBusQueue Trigger sample
```
"bindings": [
    {
      "name": "myQueueItem",
      "type": "serviceBusTrigger",
      "direction": "in",
      "queueName": "myqueue",
      "connection": "pgopafunctestsb_RootManageSharedAccessKey_SERVICEBUS",
      "accessRights": "Manage"
    }
```

Reply:
No, this is not a trigger. This is an http-triggered check to see if a trigger has fired. Similar to other bindings.